parent/pom.xml

```diff
@@ -4048,6 +4110,13 @@
                     <groupId>org.apache.camel</groupId>
                     <artifactId>camel-package-maven-plugin</artifactId>
                     <version>${project.version}</version>
+                    <dependencies>
```

Comment:
Oh maybe this jaxb dependency should be added directly to camel-package-maven-plugin so its not needed to be added in this special way

Reply:
Oh, good idea. I think we can remove this, since it's already defined as a dependency in that module.